includes/class-newspack-popups-inserter.php

```diff
@@ -623,6 +623,20 @@ public static function enqueue_scripts() {
 			return;
 		}
 
+		$script_handle = 'log-page-view';
+		\wp_register_script(
+			$script_handle,
+			plugins_url( '../dist/logPageView.js', __FILE__ ),
+			[],
+			filemtime( dirname( NEWSPACK_POPUPS_PLUGIN_FILE ) . '/dist/logPageView.js' ),
+			[
+				'strategy'  => 'defer',
+				'in_footer' => true,
+			]
+		);
+		\wp_localize_script( $script_handle, 'newspack_popups_view', [ 'donor_landing_page' => Newspack_Popups_Settings::donor_landing_page() ] );
```

Comment:
This should probably localized with a variable name other than `newspack_popups_view`, otherwise it will be overwritten by the one localized below (line 710). Probably not safe to assume that the donor landing page will always have campaigns disabled.

Reply:
Good catch, thanks! Fixed in 1a0eb2e2236fe2db7f911ae781d78dbfa89db409